Website/src/app/management/poi/client.tsx

```diff
@@ -186,14 +186,32 @@ export default function POIManagement({ poiTypes, tracks }: { poiTypes: POIType[
 								name={"selPoi"}
 								className="col-span-5 border border-gray-500 dark:bg-slate-700 rounded"
 								options={poiOptions}
+								unstyled={true}
```

Comment:
To add "dark mode" support, it was easier to remove the default style from `react-select` and define my own style. This *should* look essentially the same in "light mode", and actually usable in "dark mode".

If this works, I will bring this style to the other `react-select` components too, possibly by extracting the style into a separate component